lib/logstash/inputs/dlq.rb

```diff
@@ -0,0 +1,41 @@
+require 'logstash/namespace'
+require 'logstash/inputs/base'
+require 'logstash-input-dlq.jar'
+
+
+class LogStash::Inputs::Dlq < LogStash::Inputs::Base
+  config_name 'dlq'
+
+  default :codec, 'plain'
+
+  config :path, :validate => :path, :required => true
+  config :sincedb, :validate => :string, :required => false
+  config :commit_offsets, :validate => :boolean, :default => true
+  config :timestamp, :validate => :string, :required => false
```

Comment:
Maybe call it `from_timestamp` or `start_timestamp`. Just a `timestamp` may not be very obvious.